ci/buildspec.yml

```diff
@@ -33,7 +33,10 @@ phases:
       - $NIGHTLYBUILD
       - eval $NIGHTLYUPLOAD
 
-# local build will hang without artifacts section
 artifacts:
   files:
     - dist/*.whl
+    - model_archiver/dist/*.whl
+    - frontend/server/build/reports
+    - frontend/modelarchive/build/reports
```

Comment:
reports are being added to nightly builds , why?

Reply:
Report should be part of build artifacts. It's a good idea to keep all intermediate files for release.